### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by ashraffares**
the requirements of this project consists of making a clone of a https://thenextweb.com/ article page.

PULL RESQUEST DESCRIPTION
-navbar
-header
-section 1,2,3,4,5
-footer

### CODE REVIEW

**Review by ebeagusamuel — CHANGES_REQUESTED** [review]
# Invalid Submission :red_circle: 
##### I will be marking this PR as invalid because you do not have a custom readme. Please add a readme to your project and try to make it as professional as possible and please fix your failing linters.
slack(@ebeagusamuel)

**Comment by ashraffares** [comment]
@ebeagusamuel README.md done

File: README.md
Comment (krys2fa) [inline README.md]: - [x] [REQUIRED] You should check your closing backticks for this line. They should be the same number as the opening ones. With this done, the correct formatting should be applied.

**Review by krys2fa — CHANGES_REQUESTED** [review]
## Nicely done @ashraffares and @billodiallo  👏 
Your project displays good use of semantic HTML as well as CSS classes. 💪 
You have a professional-looking README as well. 👍 

Just a few more adjustments to complete the project. They can be seen below:
### Status: Changes Required ♻️ 

**NB:** Review was done at a screen sizes of **320px**, **768px**, **1440px**  **2560px**. Kindly state your preferred screen size if your clone displays better on your machine.

- [x] [REQUIRED] For the small version, the Navigation and sub-navigation row changes to a hamburger (bars) icon at the rightmost part (next to search icon) ([link to page](https://thenextweb.com/)). There is no hamburger icon in your project for this version. Please fix this.

- [x] [REQUIRED] All Elements should be aligned with [float](https://css-tricks.com/all-about-floats/) and [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) or [grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/align_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/)). You did not use `float`. You are to use `float` and either one of `flex` or `grid`. 

Please request for another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

**Comment by ashraffares** [comment]
@krys2fa  completed  burger menu - adding a float to footer-top - adding closing backticks to the link 
at looks good on  320px, 768px, 1440px i make changes to each on to look like thenextweb site

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on implementing all requirements and getting this project approved. :+1:

You can merge this branch into the main branch when you're ready and go on to your next project.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime**

**Pull request merged**
